recipes/rocrate/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "rocrate" %}
+{% set version = "0.13.0" %}
+{% set python_min = "3.9" %}  # Define the minimum Python version
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/rocrate-{{ version }}.tar.gz
+  sha256: 4e67606965ed7a45c845222bbd60bdd766886e17f21d8b603e0ae68c8cc71f2a
+
+build:
+  entry_points:
+    - rocrate=rocrate.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - pip
+  run:
+    - python >={{ python_min }}
+    - requests
+    - arcp ==0.2.0
```

Comment:
heads up that the required `arcp` version is 0.2.1 (see the [requirements.txt](https://github.com/ResearchObject/ro-crate-py/blob/master/requirements.txt)), I'm not sure if having an older version can cause problems when using the package

Reply:
Okay, I will update it and wait for [arcp 0.2.1](https://github.com/conda-forge/arcp-feedstock/pull/6/files) to be available.

Reply:
```suggestion
    - arcp ==0.2.1
```

Reply:
The `0.2.1` version is available now 🙂